pom.xml

```diff
@@ -643,7 +643,7 @@
             <enableProcessChecker>all</enableProcessChecker>
             <forkedProcessTimeoutInSeconds>600</forkedProcessTimeoutInSeconds>
             <!-- @argLine is filled by jacoco maven plugin. @{} means late evaluation -->
-            <argLine>-Xmx2g -XX:+HeapDumpOnOutOfMemoryError @{argLine}</argLine>
+            <argLine>-Xmx8g -XX:+HeapDumpOnOutOfMemoryError @{argLine}</argLine>
```

Comment:
When turning on the advanced reference trace, it does need more memory.  I recall that I did similar change for running with advanced reference trace.